### ISSUE TITLE
What the frak?

### ISSUE THREAD

**Issue by TyrfingMjolnir** [ogham/exa#140]
```BASh
$ uname -a
Darwin machine.local 16.3.0 Darwin Kernel Version 16.3.0: Thu Nov 17 20:23:58 PST 2016; root:xnu-3789.31.2~1/RELEASE_X86_64 x86_64

$ brew --version
Homebrew 1.1.5
Homebrew/homebrew-core (git revision 84c7; last commit 2016-12-28)

$ exa --version
exa(28617,0x7fffb36373c0) malloc: *** malloc_zone_unregister() failed for 0x7fffb362d000

$ brew cask reinstall exa
==> Downloading https://github.com/ogham/exa/releases/download/v0.4.0/exa-osx-x86_64.zip

$ /usr/local/Caskroom/exa/0.4.0/exa-osx-x86_64
exa-osx-x86_64(28430,0x7fffb36373c0) malloc: *** malloc_zone_unregister() failed for 0x7fffb362d000
```

**Comment by grenmester** [comment]
I also had this issue and I "fixed" it by installing `exa` without the `--git` option from source with `cargo build --release --no-default-features`.

**Comment by TyrfingMjolnir** [comment]
$ brew install Caskroom/cask/cargo
==> brew cask install Caskroom/cask/cargo
==> Downloading https://static.rust-lang.org/cargo-dist/cargo-nightly-x86_64-apple-darwin.tar.gz
######################################################################## 100.0%
==> No checksum defined for Cask cargo, skipping verification
==> Moving Generic Artifact 'LICENSE-APACHE' to '/usr/local/share/doc/cargo/LICENSE-APACHE'
==> Moving Generic Artifact 'LICENSE-MIT' to '/usr/local/share/doc/cargo/LICENSE-MIT'
==> Moving Generic Artifact 'LICENSE-THIRD-PARTY' to '/usr/local/share/doc/cargo/LICENSE-THIRD-PARTY'
==> Moving Generic Artifact 'README.md' to '/usr/local/share/doc/cargo/README.md'
==> Moving Generic Artifact 'cargo.1' to '/usr/local/share/man/man1/cargo.1'
==> Symlinking Binary 'cargo' to '/usr/local/bin/cargo'
🍺  cargo was successfully installed!
$ cargo build --release --no-default-features
error: could not find `Cargo.toml` in `/Users/user` or any parent directory

**Comment by grenmester** [comment]
Use `curl https://sh.rustup.rs -sSf | sh` to install the rust toolchain which will include `cargo` and `rustc`. Also, you might want to build the nightly version since that is the version of `cargo` I used to install `exa`.

**Comment by TyrfingMjolnir** [comment]
I have no need for the full cargo tool chain, I just like exa.

**Comment by grenmester** [comment]
If you want to build it from source you will need `cargo` and `rustc` which can be installed with the instructions above. If you know of any other method of installing `exa`, please share it with me.

**Comment by santagada** [comment]
I have the same problem, this binary doesn't work on macOS 10.12.3

**Comment by grenmester** [comment]
You can get it to work if you compile it without the `git` option. I'm not sure why the `git` option doesn't work with Sierra.

**Comment by iKevinY** [comment]
This can probably be closed now that 0.4.1 is installable via Homebrew.
